Sources/Linq2DynamoDb.DataContext.Tests/EntityManagementTests/Versioning/EntityVersioningTests.cs

```diff
@@ -0,0 +1,91 @@
+﻿using Linq2DynamoDb.DataContext.Tests.Entities;
+using Linq2DynamoDb.DataContext.Tests.Helpers;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+
+namespace Linq2DynamoDb.DataContext.Tests.EntityManagementTests.Versioning 
+{
+    [TestFixture]
+    class EntityVersioningTests : DataContextTestBase
+    {
+        public override void SetUp() 
+        {
+        }
+
+        public override void TearDown() 
+        {
+        }
+
+        [Test]
+        [ExpectedException(typeof(AggregateException))]
```

Comment:
is there any way we can catch expected exceptions on the line where we actually expect them?
I'm not a huge fan of catching this at the `[Test]` level. 

Especially in this case, where we want to make sure we throw from the `contextB.SubmitChanges()` and not the `contextA` call.